src/main/java/com/greenfox/poker/controller/PokerRestController.java

```diff
@@ -0,0 +1,30 @@
+package com.greenfox.poker.controller;
+import com.greenfox.poker.model.LoginRequest;
+import com.greenfox.poker.model.PokerUser;
+import com.greenfox.poker.service.UserService;
+import javax.validation.Valid;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+@org.springframework.web.bind.annotation.RestController
+public class PokerRestController {
```

Comment:
Let's call this `UserController` instead where we'll put the other user related endpoints as well.
Also use the short annotation here instead of the whole path.